### DISCUSSION TITLE
UC1628

### DISCUSSION THREAD

**Discussion by witc** [olikraus/u8g2#1452] [General]
Hi,

I am sorry for delay, but now I am finally going to try  the UC1628 driver implementation - we discussed it before in [this thread](https://github.com/olikraus/u8g2/issues/1066).
Unfortunately in [that datashee](https://github.com/olikraus/u8g2/files/3928233/UC1628c_A1.0.pdf)t is not init sequency. But I found [this ](https://www.topwaydisplay.com/sites/default/files/2020-06/LM160160R.txt)- there is some init code.

I am trying to edit the lib from uc1611 and now I can see some random points on LCD. If I progress i will let you know.

Thank you very much for any help.

Jan.

**Comment by olikraus** [comment]
I have reopened the issue
